### PR TITLE
Code: Update Donghee Na's name

### PR DESCRIPTION
**Pull request by hugovk**
Opened separately from the docs changes in https://github.com/python/cpython/pull/109743, because we can't backport non-docs things to 3.12 until after 3.12.0 is out on 2023-10-02.

### CODE REVIEW

File: Modules/expat/xmlparse.c [hunk 1]
```diff
@@ -32,7 +32,7 @@
    Copyright (c) 2019      David Loffredo <loffredo@steptools.com>
    Copyright (c) 2019-2020 Ben Wagner <bungeman@chromium.org>
    Copyright (c) 2019      Vadim Zeitlin <vadim@zeitlins.org>
-   Copyright (c) 2021      Dong-hee Na <donghee.na@python.org>
+   Copyright (c) 2021      Donghee Na <donghee.na@python.org>
```
Comment (corona10) [inline Modules/expat/xmlparse.c]: It's copied from libexpat.
https://github.com/libexpat/libexpat

I am not sure that updating legal name for expat is proper :(

Comment (corona10) [reply Modules/expat/xmlparse.c]: https://github.com/libexpat/libexpat/pull/754

I sent the PR to the upstream too :)
Let's wait

Comment (hartwork) [reply Modules/expat/xmlparse.c]: I'm positive that (1) we'll have a version of https://github.com/libexpat/libexpat/pull/754 merged upstream shortly and that (2) there will be a release >=2.5.1 this year bringing these changes to CPython "for free" just by upgrading the bundled copy of libexpat.

**Review by corona10 — APPROVED** [review]
LGTM Thank you so much!!

File: Modules/expat/xmlparse.c (hunk 1 above)
Comment (hugovk) [reply Modules/expat/xmlparse.c]: Sounds good, I'll revert the changes to `Modules/expat/` here, and we can wait for the upstream changes to filter through.

**Review by corona10 — APPROVED** [review]
LGTM again

**Pull request merged**